common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -700,3 +700,13 @@ d_darkspear = {
 		color3 = rgb { 255 231 167 }
```

Comment:
Many CoAs should be a bit higher, closer to the center I represented by a red dot.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123490290-ceb3f900-d624-11eb-98e1-1cb3f9676213.png)
![image](https://user-images.githubusercontent.com/46384992/123490385-f86d2000-d624-11eb-848a-fa0a505ee7b7.png)

</details>

Reply:
Try now